app/models/vote.rb

```diff
@@ -90,6 +90,7 @@ def associate_previous_vote
   end
 
   def age_previous_votes
+    Vote.connection.execute("SET CONSTRAINTS vote_age_per_user_per_motion DEFERRED")
```

Comment:
I think this needs to be in a database migration, rather than in application code. It seems like something that only needs to be run once on an existing database.